packages/client-sdk-nodejs/src/config/middleware/middleware.ts

```diff
@@ -34,6 +37,548 @@ export class MiddlewareMessage {
+        return `SetRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}" and value "${TEXT_DECODER.decode(request.cache_body)}" and ttl ${
+          request.ttl_milliseconds
+        } ms`;
+      }
+      case cache.cache_client._GetBatchRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._GetBatchRequest;
+        return (
+          'GetBatchRequest with keys ' +
+          request.items.reduce(
+            (acc, item) => acc + TEXT_DECODER.decode(item.cache_key) + ', ',
+            ''
+          )
+        );
+      }
+      case cache.cache_client._DeleteRequest: {
+        const request = this._grpcMessage as cache.cache_client._DeleteRequest;
+        return `DeleteRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}"`;
+      }
+      case cache.cache_client._SetBatchRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._SetBatchRequest;
+        return (
+          'SetBatchRequest with items ' +
+          request.items.reduce(
+            (acc, item) =>
+              acc +
+              `(key "${TEXT_DECODER.decode(
+                item.cache_key
+              )}" | value "${TEXT_DECODER.decode(item.cache_body)}" | ttl ${
+                item.ttl_milliseconds
+              } ms), `,
+            ''
+          )
+        );
+      }
+      case cache.cache_client._SetIfRequest: {
+        const request = this._grpcMessage as cache.cache_client._SetIfRequest;
+        return `SetIfRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}" and value "${TEXT_DECODER.decode(
+          request.cache_body
+        )}" and condition "${request.condition}"`;
+      }
+      case cache.cache_client._KeysExistRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._KeysExistRequest;
+        return (
+          'KeysExistRequest with keys ' +
+          request.cache_keys.reduce(
+            (acc, key) => acc + TEXT_DECODER.decode(key) + ', ',
+            ''
+          )
+        );
+      }
+      case cache.cache_client._IncrementRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._IncrementRequest;
+        return `IncrementRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}" and amount "${request.amount}" and ttl ${
+          request.ttl_milliseconds
+        } ms`;
+      }
+      case cache.cache_client._UpdateTtlRequest: {
+        const request = this
+          ._grpcMessage as cache.cache_client._UpdateTtlRequest;
+        return `UpdateTtlRequest with key "${TEXT_DECODER.decode(
+          request.cache_key
+        )}" and amount "${
+          String(request.increase_to_milliseconds) + ' (increase)' ||
+          String(request.decrease_to_milliseconds) + ' (decrease)' ||
+          String(request.overwrite_to_milliseconds) + ' (overwrite)'
```

Comment:
It looks to me like this will always output the first and only the first of these strings. If I run:
```
console.log(`${String(0) || 'hello' || 'world'}`);
```
it outputs `0`.